docker-compose/envs/common-stats.env

```diff
@@ -25,3 +25,5 @@ STATS__JAEGER__AGENT_ENDPOINT=localhost:6831
 
 STATS__TRACING__ENABLED=true
 STATS__TRACING__FORMAT=default
+
+STATS__IGNORE_BLOCKSCOUT_API_ABSENCE=true
```

Comment:
Are we using a newer version of the stats service than the one we tested when forked and deployed Blockscout? If so, the newer version may need additional requirements, therefore the error shown in the logs.

The version used is managed by the `STATS_DOCKER_TAG` variable in `docker-compose/services/stats.yml`.

Reply:
@gabmontes i think the image for stats is set to `:latest` as `default`... 
```
ghcr.io/blockscout/stats:latest 
```
No updates from our end other tan the actual fix for the error we got... and we applied the fix based on the information from the error log. 
```
Please specify it with corresponding env variable (`STATS__BLOCKSCOUT_API_URL`) or set `STATS__IGNORE_BLOCKSCOUT_API_ABSENCE=true` to disable functionality depending on the API.
```

cc: @max-sanchez

Reply:
Did you try setting `STATS__BLOCKSCOUT_API_URL` to `https://testnet.explorer.hemi.xyz/api`?

Reply:
@gabmontes Is this setting tested already or do you need me to test it?